Extensions/SendTo/InstructionsViewController.swift

```diff
@@ -47,6 +47,7 @@ func setupHelpView(view: UIView, introText: String, showMeText: String) {
     label1.lineBreakMode = NSLineBreakMode.ByWordWrapping
     label1.font = InstructionsViewControllerUX.TextFont
     label1.textColor = InstructionsViewControllerUX.TextColor
+    label1.textAlignment = NSTextAlignment.Center
     label1.snp_makeConstraints { (make) -> Void in
         make.width.equalTo(250)
```

Comment:
My uninformed speculation would be that this width needs to adapt, rather than aligning the text centered. Can you attach a screenshot, @sleroux?